sdks/java/io/pom.xml

```diff
@@ -32,9 +32,36 @@
   <description>Beam SDK Java IO provides different connectivity components
   (sources and sinks) to consume and produce data from systems.</description>
 
+  <properties>
+    <hadoop.version>2.7.3</hadoop.version>
```

Comment:
Can you add a comment here explaining why we have this and how it related to when the module is used in production?

I'm guessing it's something like "This is the version of Hadoop the common modules will be compiled against. However, the dependencies should be used at `provided` scope and a user's production will supply their own Hadoop version at runtime."